tgui/packages/tgui/interfaces/MobInteraction.tsx

```diff
@@ -23,41 +23,6 @@ type InteractionData = {
   type: number;
 }
 
-type GenitalInfo = {
```

Comment:
Please do not remove EVERYTHING related to the genitals in the interaction menu, only specifically the things that were added in the pregnancy pr

Reply:
done